src/main/java/org/kitteh/irc/client/library/command/KickCommand.java

```diff
@@ -43,6 +43,19 @@
     /**
      * Constructs a KICK command for a given channel.
      *
+     * @param channel channel targeted
+     * @throws IllegalArgumentException if null parameters or Channel is from
+     * another Client
+     * @see Channel#kick(User)
+     * @see Channel#kick(User, String)
+     */
+    public KickCommand(@Nonnull Channel channel) {
```

Comment:
The kick command by default doesn't give a reason. Or have a target. I might re-do to have target specified, but currently it would look like this (after some API change):

`channel.kick().target("Zarthus").reason("Because I said so").execute();`

Might go to

`channel.kick("Zarthus").reason("Yes").execute();`